libsolidity/interface/CompilerStack.cpp

```diff
@@ -1024,6 +1025,34 @@ Json::Value CompilerStack::methodIdentifiers(string const& _contractName) const
 	return methodIdentifiers;
 }
 
+Json::Value CompilerStack::errorIdentifiers(string const& _contractName) const
+{
+	if (m_stackState < AnalysisPerformed)
+		solThrow(CompilerError, "Analysis was not successful.");
+
+	Json::Value errorIdentifiers(Json::objectValue);
+	for (auto const& it: contractDefinition(_contractName).interfaceErrors()) {
+		string signature = it->functionType(true)->externalSignature();
```

Comment:
```suggestion
	for (auto const& it: contractDefinition(_contractName).interfaceErrors()) 
	{
		string signature = it->functionType(true)->externalSignature();
```

Reply:
For some reason, `./scripts/check_style.sh` didn't catch it. Is that normal?

Reply:
@joshieDo Yeah, the check style is just a basic bash script. Not surprised.